src/App.tsx

```diff
@@ -1,12 +1,78 @@
+import { FC, useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import {
+  Link,
+  NavLink,
+  Navigate,
+  Route,
+  Routes,
+  useParams,
+} from 'react-router-dom';
 
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+const tabs = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
+
+interface Tab {
+  id: string;
+  title: string;
+  content: string;
+}
+
+interface Props {
+  tabsArray: Tab[];
+  tabId: string;
+}
+
+const getTabById = (tabsArr: Tab[], idFindBy: string) => {
+  return tabsArr.find(tab => tab.id === idFindBy) || null;
+};
+
+const Tabs: FC<Props> = ({ tabsArray, tabId }) => {
+  const [selectedTab, setSelectedTab] = useState<Tab | null>(null);
+
+  useEffect(() => {
+    setSelectedTab(getTabById(tabsArray, tabId));
+  }, [tabId]);
+
+  return (
+    <>
+      <div className="tabs is-boxed">
+        <ul>
+          {tabsArray.map(tab => {
+            return (
+              <li
+                data-cy="Tab"
+                className={`${tab.id === tabId ? 'is-active' : ''}`}
+              >
+                <Link to={`/tabs/${tab.id}`}>{tab.title}</Link>
```

Comment:
If you want to make your component more universal, don't hardcode the address
```suggestion
                <Link to={`.../${tab.id}`}>{tab.title}</Link>
```

Reply:
Agree, fix it